scripts/webpack.prod.js

```diff
@@ -18,6 +18,7 @@ module.exports = merge(baseConfig, {
     library: 'Scrollbar',
     libraryTarget: 'umd',
     libraryExport: 'default',
+    globalObject: 'this'
```

Comment:
Does this require a specific version of webpack? I'm using `4.40.2` in this project now.

Reply:
yup, this is one of the webpack 4 issue [https://github.com/webpack/webpack/issues/6677](https://github.com/webpack/webpack/issues/6677)